### PR TITLE
Typos fixes.

### PR DESCRIPTION
**Pull request by jpmbandolin**
Some PHPDocs where completed to include thrown exceptions.
Some PHPDocs typos where fixed.

### CODE REVIEW

File: Slim/CallableResolver.php [hunk 1]
```diff
@@ -73,8 +75,12 @@ public function resolveMiddleware($toResolve): callable
 
     /**
      * @param string|callable $toResolve
+     * @param callable        $predicate
+     * @param string          $defaultMethod
```
Comment (theodorejb) [inline Slim/CallableResolver.php:79]: Why duplicate type information that is already in the function signature?

File: Slim/App.php [hunk 2]
```diff
@@ -98,6 +98,8 @@ public function add($middleware): self
 
     /**
      * @param MiddlewareInterface $middleware
+     *
+     * @return App
```
Comment (theodorejb) [inline Slim/App.php:102]: Why duplicate the return type info?

File: Slim/CallableResolver.php [hunk 3]
```diff
@@ -73,8 +75,12 @@ public function resolveMiddleware($toResolve): callable
...
      *
-     * @throws RuntimeException
+     * @return callable
+     * @throws ContainerExceptionInterface
```
Comment (theodorejb) [inline Slim/CallableResolver.php:82]: What is the point of duplicating the return type info and adding these `@throws` annotations?

File: Slim/CallableResolver.php [hunk 4]
```diff
@@ -114,13 +120,15 @@ private function isMiddleware($toResolve): bool
     }
 
     /**
-     * @throws RuntimeException
```
Comment (theodorejb) [inline Slim/CallableResolver.php:117]: This method does throw a `RuntimeException`. Why replace this annotation?

Also, the parameter type shouldn't be duplicated.

File: Slim/Factory/AppFactory.php [hunk 5]
```diff
@@ -63,6 +65,13 @@ public static function create(
         );
     }
 
+    /**
+     * @param ContainerInterface $container
+     *
+     * @return App
+     * @throws ContainerExceptionInterface
```
Comment (theodorejb) [inline Slim/Factory/AppFactory.php:72]: Why add `@throws` annotations which the method itself doesn't throw?

Also, the parameter and return type shouldn't be duplicated.

File: Slim/Middleware/OutputBufferingMiddleware.php [hunk 6]
```diff
@@ -41,7 +41,7 @@ public function __construct(StreamFactoryInterface $streamFactory, string $style
         $this->style = $style;
 
         if (!in_array($style, [static::APPEND, static::PREPEND], true)) {
-            throw new InvalidArgumentException("Invalid style `{$style}`. Must be `append` or `prepend`");
+            throw new InvalidArgumentException("Invalid style `$style`. Must be `append` or `prepend`");
```
Comment (theodorejb) [inline Slim/Middleware/OutputBufferingMiddleware.php:44]: Why change this?

**Review by theodorejb — CHANGES_REQUESTED** [review]

**Comment by l0gicgate** [comment]
We've removed some of the docblocs because of return types they are not needed anymore and can be inferred by IDEs. 

Can we focus on only fixing typos on this PR?

**Comment by akrabat** [comment]
Closing as no response from creator. Feel free to reopen when addressed.

**Pull request closed without merging**